android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -3744,6 +3764,40 @@ private static void addBookmark(String title, Integer pageNumber, MethodChannel.
         }
     }
 
+    private static void zoomWithCenter(MethodCall call, MethodChannel.Result result, ViewerComponent component){
+        PDFViewCtrl pdfViewCtrl = component.getPdfViewCtrl();
+        if (pdfViewCtrl == null) {
+            result.error("InvalidState", "PDFViewCtrl not found", null);
+            return;
+        }
+        double zoom = call.argument(KEY_ZOOM);
+        int x = call.argument(KEY_X);
+        int y = call.argument(KEY_Y);
+        pdfViewCtrl.setZoom(x, y, zoom);
+        result.success(null);
+    }
+
+    private static void zoomToRect(MethodCall call, MethodChannel.Result result, ViewerComponent component) throws PDFNetException{
```

Comment:
since there is a catch in this function, not sure what is throwing `PDFNetException`?